pkg/service/elasticbeanstalk/environmenthealth.go

```diff
@@ -47,6 +48,11 @@ func (fn *DescribeEnvironmentHealth) New(name string, config interface{}) ([]api
 			}
 			healthOutput, err := fn.DescribeEnvironmentHealthWithContext(ctx, &healthInput)
 			if err != nil {
+				if awsErr, ok := err.(awserr.Error); ok && awsErr.Code() == elasticbeanstalk.ErrCodeInvalidRequestException {
```

Comment:
Is there any way to determine whether an enviornment has enhanced health reporting prior to making the API call? If not, I'd rather remove this call entirely. We don't model it on the content side.

Reply:
don't think there's a way, i'll remove it